native/packages/react-native-bpk-component-card/src/BpkCard.js

```diff
@@ -57,22 +57,17 @@ const {
 const styles = StyleSheet.create({
   card: {
     backgroundColor: colorWhite,
+    borderRadius: borderRadiusSm,
     elevation: elevationXs,
     shadowColor: shadowSmColor,
     shadowOffset: { width: shadowSmOffsetWidth, height: shadowSmOffsetHeight / PixelRatio.get() },
     shadowOpacity: shadowSmOpacity,
     shadowRadius: shadowSmRadius / PixelRatio.get(),
   },
-  common: {
-    borderRadius: borderRadiusSm,
-  },
-  overflowHidden: {
```

Comment:
Tested this on iOS & android - this style seemed to be redundant. 

Popped some extra long words to try and force an overflow ... nothing happened. Happy to be proven wrong..?

Reply:
Ah actually this style mattered for the inner `<View>` and `padded={false}` as it's corners would bleed outwith the outer container. Very interesting ... Setting `color: 'transparent'` also works here... worth keeping in mind in terms of a gotcha.

cc @k0nserv, @shaundon, @matteo-hertel, @georgegillams

Reply:
We should document in code when we do things that "seem" wrong but have particular purpose

Reply:
@k0nserv agreed - we should always make it clear when we're doing something unusual as to what's happening :)